xbmc/cores/VideoPlayer/VideoPlayer.h

```diff
@@ -514,10 +514,19 @@ class CVideoPlayer : public IPlayer, public CThread, public IVideoPlayer,
   int m_demuxerSpeed = DVD_PLAYSPEED_NORMAL;
   struct SSpeedState
   {
-    double lastpts;  // holds last display pts during ff/rw operations
-    int64_t lasttime;
-    double lastseekpts;
-    double lastabstime;
+    double lastpts{0.0}; // holds last display pts during ff/rw operations
+    int64_t lasttime{0};
+    double lastseekpts{0.0};
+    double lastabstime{0.0};
+
+    void FlushBuffers(double pts)
```

Comment:
FlushBuffers might be mislieading as a name here bacause this struct does not have any buffers. Maybe just "reset"
Apart from that it looks good imo.

Reply:
Might have been mislead by your comment "reset method (with current pts) to this struct **and call it FlushBuffers too.** ". I agree Reset is better, done on the last push.
Thanks for your feedback and availability

Reply:
sorry, I forgot the "in"